Assets/MixedRealityToolkit.SDK/Inspectors/UX/Interactable/InteractableInspector.cs

```diff
@@ -58,7 +58,7 @@ protected virtual void OnEnable()
             isGlobal = serializedObject.FindProperty("isGlobal");
             canSelect = serializedObject.FindProperty("CanSelect");
             canDeselect = serializedObject.FindProperty("CanDeselect");
-            startDimensionIndex = serializedObject.FindProperty("StartDimensionIndex");
+            startDimensionIndex = serializedObject.FindProperty("startDimensionIndex");
```

Comment:
we should mark the property as being previously serialized as the old name

Reply:
Looks like it's marked in the original PR: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/6104/files#diff-a34c3fc73243420b0de3150d786c43e4R196, just not accounted for in the inspector

Reply:
It was marked there. But the inspector script is trying to look for the old property name and is not accounted for here.